courses/templates/subscription.html

```diff
@@ -0,0 +1,15 @@
+<div class="posted-block">
+    <div class="container">
+        <h1>WANT TO LEARN ABOUT NEW COURSES FROM MIT XPRO?</h1>
```

Comment:
Could you please switch it to title case and add the `text-uppercase` class to the heading?

Reply:
Not necessary as It is hard code inside a template.